src/handlers/scheduledEventHandler.ts

```diff
@@ -0,0 +1,54 @@
+import { env } from "../typeDefinitions/default.types";
+import { taskOverDueDiscordMembers } from "../utils/taskOverDueDiscordMembers";
+import * as error from "../constants/responses";
+import { getDiscordIds } from "../utils/getDiscordIds";
+import { RDS_TRACKING_CHANNEL_URL } from "../constants/urls";
+
+export async function send(env: env): Promise<void> {
+  try {
+    const assigneeIds: string[] | string = await taskOverDueDiscordMembers();
+
+    //A user might have more than one task which are running red
+    //so to mention them just once, we are using Set to filter out
+    const discordIds: string[] | string = await getDiscordIds(assigneeIds);
+    const uniqueDiscordIds = [...new Set(discordIds)];
+
+    //notifying the two users with the authority.
+    let str = `<@154585730465660929> <@1040700289348542566>\nThese people have their task running red:\n`;
```

Comment:
I guess hard-coding these `userIds` makes understanding the code a bit more difficult.